.github/workflows/automerge.yml

```diff
@@ -22,24 +22,30 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: | 
+        set -e

```

Comment:
I think the `if then exit` pattern is much safer. That way, we can at least `echo` something before exiting instead of failing silently.